pkg/handler/repositories.go

```diff
@@ -318,3 +325,70 @@ func (rh *RepositoryHandler) deleteRepository(c echo.Context) error {
 	}
 	return c.NoContent(http.StatusNoContent)
 }
+
+// IntrospectRepository godoc
+// @summary 		introspect a repository
+// @ID				introspect
+// @Tags			repositories
+// @Param  			uuid            path    string  true  "Identifier of the Repository"
+// @Param			reset_count     query   bool    false "Reset the failed introspections count"
```

Comment:
any reason this is a query param instead of part of the body?

Reply:
I guess not. I'll make it part of the body,